Tree/PhpcrOdmTree.php

```diff
@@ -127,6 +127,23 @@ public function __construct(
 
         $this->depth = $options['depth'];
         $this->preciseChildren = $options['precise_children'];
+
+        if (!$this->assetHelper instanceof CoreAssetsHelper) {
+            // check for the AssetsHelper class introduced in Symfony 2.7
+            if (!$this->assetHelper instanceof \Symfony\Bundle\FrameworkBundle\Templating\Helper\AssetsHelper) {
```

Comment:
cool! please add a `use` statement for this class. this will not produce an error even if the class is not available. (sometimes the sloppiness of php is quite nice :-) )